# HG changeset patch
# User "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
# Date 1322160515 -3600
# Node ID cac189eafb049bc7a3a07771ac9dfd5a516d80ba
# Parent  ac0db1b6e9d7accf7353d8ea4589cf212c7005c1
arch/sparc: Sparc supports setting the floating point type

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>

diff --git a/config/arch/sparc.in b/config/arch/sparc.in
--- a/config/arch/sparc.in
+++ b/config/arch/sparc.in
@@ -6,6 +6,7 @@
 ## select ARCH_USE_MMU
 ## select ARCH_SUPPORTS_WITH_CPU
 ## select ARCH_SUPPORTS_WITH_TUNE
+## select ARCH_SUPPORTS_WITH_FLOAT
 ##
 ## help The SUN SPARC architecture, as defined by:
 ## help     32 bit: http://www.sparc.org/standards/V8.pdf
